eeps/eep-0072.md

```diff
@@ -0,0 +1,145 @@
+    Author: Jesse Gumm <ja(at)gumm(dot)io>
+    Status: Draft
+    Type: Standards Track
+    Created: 08-Oct-2024
+    Post-History:
+****
+EEP 72: Reserved words and Variables as record names, and enhancement to definition syntax
+----
+
+Abstract
+========
+
+This EEP loosens some of the restrictions around record names such no longer is
+it necessary to quote record names when those records are named with reserved
+words (`#if` vs `#'if'`) or words with capitalized first characters (`#Hello`
+vs `#'Hello'`).
```

Comment:
```suggestion
words (`#if` vs `#'if'`) or words with capitalized first characters (variable
names, for example `#Hello`￼vs. `#'Hello'`).
```

Reply:
You have a point. Maybe just "like variable names".  I interpret "atom with variable capitalization" as `'Hello'` because quoting is how to create such an atom.
We should avoid inventing an acronym...

Reply:
I've updated the phrasing here: https://github.com/erlang/eep/pull/64/commits/dbfb39e626bc930f6378af45c1375c2599955edb

What do you think?